### PR TITLE
Format TWP18 and add test

### PR DESCRIPTION
**Pull request by LeoCumpli21**
## Summary

This PR closes LeoCumpli21/PyZombis#32 by formatting all code with `black`. Also, I added a web test for activecode 5.
*Note*: I didn't focus on redaction nor grammar.

## Checklist

- [x] Variables, functions and comments are translated to Spanish
- [x] Functions follow underscore notation
- [x] Spell check done & typos fixed
- [x] All python code is PEP8 compliant
- [x] Test coverage with Playwright implemented; locators are Pyhton code
- [x] Reviewers assigned (all peers & at least 1 mentor)

## Screenshots

https://user-images.githubusercontent.com/80920858/122971114-4a2b5700-d354-11eb-8cf6-8f34cc8c1a64.mp4

### CODE REVIEW

File: _sources/lectures/TWP18.rst [hunk 1]
```diff
@@ -4,91 +4,82 @@ Strings
...
-        <html>
-               <head>
-                  <title> Teste </title>
-               </head>
-               <body>
-                  <p>Testando</p>
-               </body>
-         <html>'''
-
+    x = "aguacate"
```
Comment (angelasofiaremolinagutierrez) [inline _sources/lectures/TWP18.rst:21]:
```suggestion
    x = 'aguacate'
```
The exercise says:
> ¿Puedo usar comillas simples, dobles o triples?

I think there are three `print()` functions, each one to show each kind of possibility to make a string: `'` `"` `"""`.

Comment (LeoCumpli21) [reply _sources/lectures/TWP18.rst:21]: Right! Thanks.

File: _sources/lectures/TWP18.rst [hunk 2]
```diff
@@ -97,224 +88,242 @@ Incremento en el corte
...
+    :nocodelens:
+    :stdin:
    
-   ====
-   from unittest.gui import TestCaseGui
+    Haga un programa que solicite la fecha de nacimiento en formato "dd/mm/aaaa" y 
+    convierta esta fecha a formato "<día> de <mes> de <año>" usando la lista ``meses`` 
+    que ya está escrita. Guardar el resultado en la variable ``fecha_de_nacimiento`` e 
+    imprimir esta variable. Recordar que ``.split()`` regresa una lista y se le puede pasar 
+    como argumento el caracter con el cual separar un string. Guardar el día, mes y el año
```
Comment (y-martinez) [inline _sources/lectures/TWP18.rst:240]:
```suggestion
    como argumento el caracter con el cual separar un string. Guardar el día, mes y el año en formato de cadena 
```

File: _sources/lectures/TWP18.rst [hunk 3]
```diff
@@ -97,224 +88,242 @@ Incremento en el corte
...
+    texto = "@" + texto[1:]
+    print(texto)
 
 
 Concentración
 =============
 
 
-
 + Ejemplo de un programa que lee una palabra y reemplaza las vocales con "*"
```
Comment (y-martinez) [inline _sources/lectures/TWP18.rst:135]:
```suggestion
+ Ejemplo de un programa que lee una palabra y reemplaza las vocales por ``"*"``
```

**Review by y-martinez — APPROVED** [review]
I left some suggestions but In general LGTM,

**Review by NicolasSandoval — CHANGES_REQUESTED** [review]
Nice work, but in the 'Concentración' exercises capital letters are not considered, maybe you can add the `lower()` function to the words or add in the text that only works with lowercase.

**Comment by LeoCumpli21** [comment]
> Nice work, but in the 'Concentración' exercises capital letters are not considered, maybe you can add the `lower()` function to the words or add in the text that only works with lowercase.

Done @NicolasSandoval 😄

**Review by NicolasSandoval — APPROVED** [review]
Great! Thanks.

**Pull request merged**